addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -248,10 +248,40 @@ def migration_invoice_moves(env):
         RETURNING aml.id""",
     )
     aml_ids = tuple(x[0] for x in env.cr.fetchall())
+    # Fix (exceptional) cases where two or more move lines points to the same old invoice line
+    while True:
```

Comment:
Shouldn't this loop be in a more wider scope, for applying all possible matches each time?

Reply:
ah, you mean to include the query of relaxed criteria here, right?